wagtail/images/views/chooser.py

```diff
@@ -164,12 +172,21 @@ def chooser_upload(request):
         form = ImageForm(user=request.user)
 
     images = Image.objects.order_by('-created_at')
+
+    # allow hooks to modify the queryset
+    for hook in hooks.get_hooks('construct_image_chooser_queryset'):
+        images = hook(images, request)
```

Comment:
Two birds in one stone (as far as I understand this wasn't causing the bug but was missing regardless) 👍 

It might be pushing it a bit too far (since it shouldn't change really, there isn't many ways to apply the hooks) but it would almost deserve its own `get_chooser_images` function.